data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/worker/client/ElasticsearchAccessor.java

```diff
@@ -56,4 +56,9 @@ public SearchScrollResponse searchWithScroll(SearchScrollRequest searchScrollReq
     public void deleteScroll(DeleteScrollRequest deleteScrollRequest) {
         //todo: implement
     }
+
+    @Override
+    public Object getClient() {
```

Comment:
Why do we implement `ClusterClientFactory` here if we are returning null?

Reply:
The ElasticSearch accessor is not implemented yet